frontend/Context.tsx

```diff
@@ -0,0 +1,144 @@
+// Context.tsx
+import React, { createContext, useContext, useState, ReactNode } from "react";
+
+interface FormProviderProps {
+  children: ReactNode;
+}
+
+interface SchoolsAttended {
+  schoolName: string;
+  schoolCity: string;
+  schoolState: string;
+  schoolCountry: string;
+  coreUnits: string;
+  degreeReceived: string;
+  dateStarted: string;
+  dateStopped: string;
+}
+interface ProfessionalMemberships {
+  membershipName: string;
+  membershipLevel: string;
+}
+interface ICCCourses {
+  courseName: string;
+  courseCompleteDate: string;
+}
+interface NationalExams {
+  examName: string;
+  examDate: string;
+  certificateNumer: string;
+}
+
+interface WorkExperience {
+  designExperience: string;
+  numHours: string;
```

Comment:
What do you think about numHours (or any other field that takes a number) being of type number rather than string?